src/mlpack/core/tree/cover_tree/dual_tree_traverser.hpp

```diff
@@ -87,19 +87,23 @@ class CoverTree<MetricType, StatisticType, MatType, RootPointPolicy>::
    * Helper function for traversal of the two trees.
    */
   void Traverse(CoverTree& queryNode,
-                std::map<int, std::vector<DualCoverTreeMapEntry> >&
-                    referenceMap);
+                std::map<int,
+                         std::vector<DualCoverTreeMapEntry>,
+                         std::greater<int>>& referenceMap);
```

Comment:
```suggestion
      std::map<int, std::vector<DualCoverTreeMapEntry>,
          std::greater<int>>& referenceMap);
```
This one is a bit tough; the std::map type is really long.  But maybe this is better?